src/coreclr/jit/emitarm64.cpp

```diff
@@ -15681,6 +15750,14 @@ emitter::insExecutionCharacteristics emitter::getInsExecutionCharacteristics(ins
             }
             break;
 
+        // TODO-PAC: Update the latency and throughput values appropriately.
```

Comment:
once this is fixed, this PR should be good to go. Thanks!

Reply:
Done.